cmd/tk/util.go

```diff
@@ -15,31 +15,36 @@ func pageln(i ...interface{}) {
 	fPageln(strings.NewReader(fmt.Sprint(i...)))
 }
 
-// fPageln invokes the systems pager with the supplied data
-// falls back to fmt.Println() when paging fails or non-interactive
+// fPageln invokes the systems pager with the supplied data.
+// If the PAGER environment variable is empty, no pager is used.
+// If the PAGER environment variable is unset, use less with posix flags.
 func fPageln(r io.Reader) {
-	// get system pager, fallback to `less`
-	pager := os.Getenv("PAGER")
-	var args []string
-	if pager == "" || pager == "less" {
+	pagerEnv, ok := os.LookupEnv("PAGER")
+	if !ok {
 		// --RAW-CONTROL-CHARS  Honors colors from diff. Must be in all caps, otherwise display issues occur.
 		// --quit-if-one-screen Closer to the git experience.
 		// --no-init            Don't clear the screen when exiting.
-		pager = "less"
-		args = []string{"--RAW-CONTROL-CHARS", "--quit-if-one-screen", "--no-init"}
+		pagerEnv = "less --RAW-CONTROL-CHARS --quit-if-one-screen--no-init"
```

Comment:
```suggestion
		pagerEnv = "less --RAW-CONTROL-CHARS --quit-if-one-screen --no-init"
```

Above also aren't Busybox compatible, but this is fine for me as we now have a way around it.

```
BusyBox v1.31.1 () multi-call binary.

Usage: less [-EFIMmNSRh~] [FILE]...
```

Are you aware what `less` implementations are around? (GNU, BSD, Busybox, etc)?

Reply:
I believe that for BSD distributions (at least freeBSD) `less` is the same GNU less as in Linux. Busybox has it's own implementation but I think we are agreed that providing the user the possibility to override is preferable to supporting the lowest common denominator (which would probably be `more` I suppose).

Reply:
Totally support that! more seems to lack usable color support, which I consider essential here. 

Running perfect with GNU less should probably be our aim here, while giving people a way to use any other tool / implementation as needed.